backend/prisma/migrations/20220930121151_add_user_and_activity_logs_relationship/migration.sql

```diff
@@ -0,0 +1,12 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `username` on the `ActivityLog` table. All the data in the column will be lost.
+  - Made the column `entityId` on table `ActivityLog` required. This step will fail if there are existing NULL values in that column.
+  - Made the column `userId` on table `ActivityLog` required. This step will fail if there are existing NULL values in that column.
+
+*/
+-- AlterTable
+ALTER TABLE "ActivityLog" DROP COLUMN "username",
```

Comment:
removed the field `username` and replaced it with a relationship between the tables `User` and `ActivityLog`